src/locale/messages.de.xlf

```diff
@@ -1363,7 +1363,7 @@
       </trans-unit>
       <trans-unit id="order.public" datatype="html">
         <source>Public</source>
-        <target>Öffentliches Mandat</target>
+        <target>Bestellung GeoGR</target>
```

Comment:
We should avoid having GeoGR specific labels in the OSS project (https://github.com/camptocamp/geoshop-front)